Streetcode/Streetcode.BLL/Attributes/Authentication/StrongPasswordAttribute.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Streetcode.BLL.Attributes.Authentication
+{
+    public class StrongPasswordAttribute : ValidationAttribute
+    {
+        protected override ValidationResult? IsValid(object? value, ValidationContext validationContext)

```

Comment:
We cannot use `string? password` here because, in such case, we won`t be able to override this method.
P.S. 
![image](https://github.com/ita-social-projects/StreetCode/assets/113620149/da52fca4-78d9-4bdb-98c3-e6ac0af34345)